src/App.jsx

```diff
@@ -228,6 +239,100 @@ function App() {
         )}
       </div>
 
+      <Dialog
+        sx={{ borderRadius: "100px" }}
+        open={openShareModal}
+        onClose={() => setOpenShareModal(false)}
+      >
+        <div
+          style={{
+            backgroundColor: "var(--bg-color)",
+            padding: "20px",
+            textAlign: "center",
+            color: "var(--color)",
+            border: "2px solid var(--color)",
+          }}
+        >
+          <img
+            src="https://user-images.githubusercontent.com/27727921/185767526-a002a17d-c12e-4a6a-82a4-dd1a13a5ecda.png"
+            alt="instagram"
+            className="modal__signup__img"
+            style={{ width: "40%", filter: "invert(var(--val))" }}
+          />
+          <p
+            style={{
+              fontSize: "1rem",
+              color: "var(--color)",
+            }}
+          >
+            Share Post
+          </p>
+
+          <div className="mx-2">
```

Comment:
icons are not dark mode proof. Please fix them

Reply:
Sure, and also I would like to create separate components. Let me create the issue after it and will ask for an assignment.

Reply:
right, it'll be of great help!